docs/docs/upgrading-node-js.md

```diff
@@ -16,18 +16,18 @@ Run `node -v` in a terminal to see which version of Node.js you have.
 
 ```shell
 node -v
-v10.18.0
+v12.21.0
 ```
 
-This example shows Node.js version 10, specifically v10.18.0.
+This example shows Node.js version 12, specifically v12.21.0.
 
-## Upgrading from Node.js version 8
+## Upgrading from Node.js version 10
 
-Node.js version 8 reached _End-of-life_ status on December 31, 2019. Many of Gatsby's dependencies are updating to Node.js version 10 and above. Gatsby must also update in order to deliver new features and bug fixes more quickly.
+Node.js version 10 reached _End-of-life_ status on April 30, 2021. Many of Gatsby's dependencies are updating to Node.js version 12 and above. Gatsby must also update in order to deliver new features and bug fixes more quickly.
```

Comment:
Node 10 didn't reach end of life yet (it still have almost 2 months left)

Reply:
I was pragmatic here. Rather than writing `reaches` and forgetting about it, I kept it `reached` and it's slightly incorrect for 2 months. But we can rephrase it:

```suggestion
Node.js version 10 _End-of-life_ date is April 30, 2021. Many of Gatsby's dependencies are updating to Node.js version 12 and above. Gatsby must also update in order to deliver new features and bug fixes more quickly.
```

Reply:
@pieh done 👍